### PR TITLE
Add Baroni-2011-200 Data on semantic relations

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

This PR is connected to #27. The list includes 200 concrete nouns and offers information on a variety of semantic relations.

### CODE REVIEW

**Comment by AnnikaTjuka** [comment]
@LinguList Should we use the normal review process for this list? The mapping was fairly straightforward because the list includes only concrete nouns. Maybe one reviewer is sufficient?

**Comment by AnnikaTjuka** [comment]
Yes, sorry, I forgot to mention this. I ran into trouble because one column was named "ATTRIBUTES" and then only tested a few alternatives with the separator. I always got the error:
`TYPICAL_ATTRIBUTE: invalid lexical value for string:  aquatic` and couldn't figure out what the problem with this word was.

**Comment by AnnikaTjuka** [comment]
I tried
```
                    {
                        "name": "TYPICAL_ATTRIBUTE",
                        "separator": ",",
                        "datatype": {
                            "base": "string",
                            "format": "([\\S]+)*"
                        }
                    },
```
And 
```
                    {
                        "name": "TYPICAL_ATTRIBUTE",
                        "separator": ",",
                        "datatype": {
                            "base": "string",
                            "format": "([\\S]+)( [\\S]+)*"
                        }
                    },
```

**Comment by AnnikaTjuka** [comment]
I found this website (https://regexr.com/) and tried several other options (also tested what I'll find in the .tsv file with Regex in Sublime). This one finds all the words `([a-z]+)( [a-z]+)*`

But I still get the same error.

**Comment by AnnikaTjuka** [comment]
Error message: 
```
INFO    concepticon/concepticon-data at /Users/tjuka/GitHub/Repos/concepticon-data
testing 328 concept lists
/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/api.py:299: Warning: WARNING:conceptlists.tsv: no PDF found for Urban2016
  warnings.warn(_msg('warning', msg, name, line), Warning)
/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/api.py:299: Warning: WARNING:conceptlists.tsv: no PDF found for Wu2020
  warnings.warn(_msg('warning', msg, name, line), Warning)
/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/api.py:299: Warning: WARNING:conceptlists.tsv: no PDF found for Goddard2011
  warnings.warn(_msg('warning', msg, name, line), Warning)
Traceback (most recent call last):
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 696, in iterdicts
    res[col.header] = col.read(v)
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 413, in read
    return [datatype.read(vv) for vv in v]
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 413, in <listcomp>
    return [datatype.read(vv) for vv in v]
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 326, in read
    return self.validate(self.parse(v))
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 303, in parse
    return self.basetype.to_python(v, **self.derived_description)
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/datatypes.py", line 73, in to_python
    string.value_error(v)
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/datatypes.py", line 40, in value_error
    raise ValueError('invalid lexical value for {}: {}'.format(cls.name, v))
ValueError: invalid lexical value for string:  aquatic

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/bin/concepticon", line 11, in <module>
    load_entry_point('pyconcepticon', 'console_scripts', 'concepticon')()
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/__main__.py", line 70, in main
    return args.main(args) or 0
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/commands/test.py", line 21, in run
    if args.repos.check(*args.clids):
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/api.py", line 429, in check
    for i, concept in enumerate(cl.concepts.values()):
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/clldutils/misc.py", line 195, in __get__
    result = instance.__dict__[self.__name__] = self.fget(instance)
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/pyconcepticon/models.py", line 249, in concepts
    for item in self.metadata:
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 698, in iterdicts
    log_or_raise(
  File "/Users/tjuka/GitHub/VirtualEnv/base-env/lib/python3.8/site-packages/csvw/metadata.py", line 48, in log_or_raise
    raise exception_cls(msg)
ValueError: /Users/tjuka/GitHub/Repos/concepticon-data/concepticondata/conceptlists/Baroni-2011-200.tsv:2:7 TYPICAL_ATTRIBUTE: invalid lexical value for string:  aquatic
```

**Comment by LinguList** [comment]
Did you just try:

"separator": ", ",
"datatype": {"base": "string"}

?

You do not need the other specification. Any string, but separator is ",
" (not "," in your case).

**Comment by AnnikaTjuka** [comment]
A classical case of unnecessarily complicating things. Your solution worked :) Thanks!

File: concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+                        "datatype": "string",
+                        "name": "ENGLISH"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "SEMANTIC_CLASS"
+                    },
+                    {
+                        "name": "TYPICAL_ATTRIBUTE",
+                        "separator": ",",
```
Comment (LinguList) [inline concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json:52]: Separator is `", "` so it is `,` = comma + ` ` space. 

BTW, try to load the concepticon and you can see what this means for loading the list:

```python
from pyconcepticon import Concepticon
con = Concepticon()
for c in con.conceptlists['Baroni-2011-200'].concepts.values():
    print(c.english, len(c.attributes['cohyponymy']), c.attributes['cohyponymy'][0])
```

If separator works fine, you should see a full word, plus the length of the items in the list.

Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json]: I added the comma + ` ` space in the metadata.json, but the script only returns the first item of each column.

Comment (LinguList) [reply concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json]: First *item* means what? First letter, or first word?

Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Baroni-2011-200.tsv-metadata.json]: First word. For the column TYPICAL_ATTRIBUTE it looks like this
```
alligator 16 aggressive
frog 14 amphibious
lizard 10 brown
snake 24 aggressive
turtle 9 amphibious
dishwasher 14 clean
freezer 17 big
fridge 17 big
oven 12 clean
phone 19 big
radio 16 big
stereo 17 big
stove 12 clean
television 21 addictive
toaster 10 clean
washer 16 big
crow 7 black
dove 5 domestic
eagle 10 aggressive
falcon 8 aggressive
goose 6 aquatic
hawk 10 aggressive
owl 9 brown
penguin 10 Antarctic
pheasant 8 brown
pigeon 7 dirty
robin 7 common
sparrow 6 brown
swan 10 aquatic
```

**Comment by LinguList** [comment]
But that's perfect! I mean: the code slices by index [0], right? That's
what I wanted to see :)

**Comment by LinguList** [comment]
So this can be merged, and we may want to check if some lists have a
similar structure (I did not find any other separator, but I am sure
there are some lists needing them). Yet it is enough to keep one's eyes
open.

**Comment by AnnikaTjuka** [comment]
Ah yes, if I add `[0:4]`, I get the first four words of a column.

**Comment by LinguList** [comment]
Yep, I just didn't want to see all words on the screen (in fact, I
didn't even try).

**Comment by AnnikaTjuka** [comment]
And I was confused :)
I was waiting to finally see `aquatic` which is the second item in the TYPICAL_ATTRIBUTE column and was returned as a key error while running `concepticon test`.

**Comment by LinguList** [comment]
I think that was the problem: you did not separate by `, ` with space,
so it splits `abc, aquatic` first to: `abc` and ` aquatic` and then uses
the regular expression to check if the condition of a space separating
the word inside the separator was met, but this condition fails, since
you have a word starting with space, which is against the rules.

**Comment by AnnikaTjuka** [comment]
Yes, that was the original problem. I also didn't pay attention to the `[0]` in the print function. So I expected that the script would return a list of all words in a given column.

**Pull request merged**